lib/views/auth/login_view.dart

```diff
@@ -242,12 +237,8 @@ class LoginScreenState extends State<LoginScreen> {
                             InkWell(
                               onTap: () {
                                 // ignore: always_specify_types
-                                Navigator.pushReplacement(context,
-                                    MaterialPageRoute<dynamic>(
-                                        builder: (BuildContext context) {
-                                  debugPrint('Navigate to sign up page');
-                                  return SignUpScreen();
-                                }));
+                                Navigator.of(context)
+                                    .pushNamed(RouteConstant.SIGN_UP_SCREEN);
```

Comment:
Use pushReplacementNamed here

Reply:
Well, I think we should not close the app if user presses back button on the sign up page.

Reply:
because of don't have a "login instead" button on the sign up screen

Reply:
Okay, then let it be, I thought we have a toggle view in between the screens.